akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ClusterSharding.scala

```diff
@@ -41,81 +41,90 @@ import akka.util.ByteString
- * typically have persistent (durable) state, but this feature is not limited to
- * actors with persistent state.
+ * more resources (e.g. memory) than fit on one machine.
+ *   - Distribution: You need to distribute them across several nodes in the cluster
+ *   - Location Transparency: You need to interact with them using their logical identifier,
+ * without having to care about their physical location in the cluster, which can change over time.
  *
- * In this context sharding means that actors with an identifier, so called entities,
- * can be automatically distributed across multiple nodes in the cluster. Each entity
- * actor runs only at one place, and messages can be sent to the entity without requiring
- * the sender to know the location of the destination actor. This is achieved by sending
- * the messages via a [[ShardRegion]] actor provided by this extension, which knows how
- * to route the message with the entity id to the final destination.
+ * '''Entities''':
+ * It could for example be actors representing Aggregate Roots in Domain-Driven Design
+ * terminology. Here we call these actors "entities" which typically have persistent
+ * (durable) state, but this feature is not limited to persistent state actors.
  *
- * This extension is supposed to be used by first, typically at system startup on each node
- * in the cluster, registering the supported entity types with the [[ClusterSharding#start]]
- * method and then the `ShardRegion` actor for a named entity type can be retrieved with
- * [[ClusterSharding#shardRegion]]. Messages to the entities are always sent via the local
- * `ShardRegion`. Some settings can be configured as described in the `akka.cluster.sharding`
- * section of the `reference.conf`.
+ * '''Sharding''':
+ * In this context sharding means that actors with an identifier, or entities,
+ * can be automatically distributed across multiple nodes in the cluster.
  *
+ * '''ShardRegion''':
+ * Each entity actor runs only at one place, and messages can be sent to the entity without
+ * requiring the sender to know the location of the destination actor. This is achieved by
+ * sending the messages via a [[ShardRegion]] actor, provided by this extension. The [[ShardRegion]]
+ * knows the shard mappings and routes inbound messages to the entity with the entity id.
+ * Messages to the entities are always sent via the local `ShardRegion`.
  * The `ShardRegion` actor is started on each node in the cluster, or group of nodes
  * tagged with a specific role. The `ShardRegion` is created with two application specific
  * functions to extract the entity identifier and the shard identifier from incoming messages.
+ *
+ * Typical usage of this extension:
+ *   1. At system startup on each cluster node by registering the supported entity types with
+ * the [[ClusterSharding#start]] method
+ *   2. Retrieve the `ShardRegion` actor for a named entity type with [[ClusterSharding#shardRegion]]
```

Comment:
Hmm this doesn't seem to be picked up correctly - '1' and '2' are one the same line for me 
![DeepinScreenshot_select-area_20190327164412](https://user-images.githubusercontent.com/131856/55090602-b5bbe680-50af-11e9-9cb3-a3d8bbb31849.png)

Reply:
Mysterious. I literally grabbed it from the Scaladoc guide with the spacing. Thanks so much for the catch.

Reply:
OIC what I did wrong. Pushing fix.